### PR TITLE
Opee drop python2 support

### PR DESCRIPTION
**Pull request by liuming50**

### CODE REVIEW

**Comment by ricardosalveti** [comment]
Great, thanks for the cleanup, giving it a test locally now.

File: meta-lmp-base/recipes-security/optee/optee-os/0001-scripts-convert-remainging-scripts-to-python3.patch [hunk 1]
```diff
@@ -0,0 +1,227 @@
+From bbaeed4dc6258006e846543197b8aff95d80abbf Mon Sep 17 00:00:00 2001
```
Comment (mike-scott) [inline meta-lmp-base/recipes-security/optee/optee-os/0001-scripts-convert-remainging-scripts-to-python3.patch]: We're using a version of OP-TEE that already has python3 compliant scripts.  I don't think this additional patch is needed.

Comment (ricardosalveti) [reply meta-lmp-base/recipes-security/optee/optee-os/0001-scripts-convert-remainging-scripts-to-python3.patch]: Not every script was converted (based on the hash we used), so we might indeed need the patch to make sure every python script is compatible with python 3.

Comment (mike-scott) [reply meta-lmp-base/recipes-security/optee/optee-os/0001-scripts-convert-remainging-scripts-to-python3.patch]: You are quite right.   I was looking at the wrong hash.  We do need this.  Apologies.

**Pull request merged**